components/apimgt/org.wso2.carbon.apimgt.common.analytics/src/main/java/org/wso2/carbon/apimgt/common/analytics/publishers/dto/ExtendedAPI.java

```diff
@@ -1,5 +1,5 @@
 /*
- *  Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *  Copyright (c) 2023, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Keep them as it is. We are adding this only if it is a new file. Continue same for the rest of the PRs

Reply:
Resolved with - 0c8ef62986423a748b571d7353abe7e39992af7d